app/helpers/validation_helper.rb

```diff
@@ -234,16 +234,16 @@ def self.normalize_enums(enums_dict)
     lab_type: { label: 'Lab Test Type', checks: [:enum] },
     specimen_collection: { label: 'Lab Specimen Collection Date', checks: [:date] },
     report: { label: 'Lab Report Date', checks: [:date] },
-    result: { label: 'Result', checks: [:enum] },
+    result: { label: 'Lab Result', checks: [:enum] },
     assigned_user: { label: 'Assigned User', checks: [] },
     continuous_exposure: { label: 'Continuous Exposure', checks: [:bool] },
     patient_id: { label: 'Patient ID', checks: [] },
     contact_attempts: { label: 'Contact Attempts', checks: [] },
     group_name: { label: 'Vaccine Group', checks: [:enum] },
-    product_name: { label: 'Product Name', checks: [:enum] },
-    administration_date: { label: 'Administration Date', checks: [:date] },
-    dose_number: { label: 'Dose Number', checks: [:enum] },
-    notes: { label: 'Notes', checks: [] },
+    product_name: { label: 'Vaccine Product Name', checks: [:enum] },
```

Comment:
`VaccineControllerTest:L351` needs to be updated to take into account this language change

Reply:
As well as `VaccineControllerTest:L277`

Reply:
https://github.com/SaraAlert/SaraAlert/pull/991/files#diff-39fc5612ff2f8959fbfa9bd906b57810f7d0d7ef161bb574d2a21ad503d3af4fR277